src/flamenco/runtime/fd_runtime.c

```diff
@@ -876,6 +876,23 @@ fd_runtime_prepare_txns_phase2_tpool( fd_exec_slot_ctx_t * slot_ctx,
         }
       }
 
+      fd_txncache_query_t curr_query;
+      curr_query.blockhash = blockhash->uc;
+      fd_blake3_t b3[1];
+      uchar hash[32];
+      fd_blake3_init( b3 );
+      fd_blake3_append( b3, ((uchar *)txn_ctx->_txn_raw->raw + txn_ctx->txn_descriptor->message_off), txn_ctx->_txn_raw->txn_sz - txn_ctx->txn_descriptor->message_off );
+      fd_blake3_fini( b3, hash );
```

Comment:
Prefer fd_blake3_hash. The streamlined function is faster in the Blake3 rewrite

Reply:
that api isn't available yet